src/ebfloeseg/preprocess.py

```diff
@@ -243,18 +244,20 @@ def _preprocess(
 
     # saving the label floes tif
     fname = "final.tif"
+    assert output.min() >= 0
```

Comment:
What is this assert statement for? Did you forget to delete it or did you intend to throw an exception?

Reply:
I intend to throw an exception if this isn't fulfilled – it should never occur, but it'll result in errors if it does.